### DISCUSSION TITLE
installing cactus

### DISCUSSION THREAD

**Discussion by ali-o-othman** [ComparativeGenomicsToolkit/cactus#1553] [Q&A]
Hello, I need some help from anyone who managed to install cactus.
 I'm following the instructions correctly but everything fails with (make -j 8) step.
 Note the problem might be that i dont have docker installed on the server and i dont have sudo rights, i can use signularity but I'm not sure how.

**Comment by glennhickey** [comment]
There's no reason to run `make`.  If you need to run on a cluster, download the binaries and install as described here https://github.com/ComparativeGenomicsToolkit/cactus/releases.  

If you don't need to run on a cluster, just run directly from the singularity using `docker://quay.io/comparative-genomics-toolkit/cactus:v2.9.3` for the image.

**Reply by ali-o-othman** [reply]
thank you for your reply!
I tried to pull docker image with this comand:

singularity pull docker://quay.io/comparative-genomics-toolkit/cactus:v2.9.3

this created a .sif file
then I tried to run cactus-hal2maf as following:

singularity exec --fakeroot cactus_v2.9.3.sif cactus-hal2maf . 447-mammalian-2022v1.hal rodent.maf --chunkSize 500000 --refGenome Mus_musculus --targetGenomes Mus_musculus,Mus_spretus,Mus_caroli,Mus_pahari,Rattus_norvegicus,Meriones_unguiculatus,Psammomys_obesus,Acomys_cahirinus --restart
 
but I got this error:
toil.jobStores.abstractJobStore.NoSuchFileException: File 'config.pickle' does not exist.

do you think this problem migh be in istallation ?
